src/api/users/tools/migrate/migrate.js

```diff
@@ -0,0 +1,68 @@
+const fetch = require('node-fetch');
+const jsdom = require('jsdom');
+const { isWebUri } = require('valid-url');
+const fs = require('fs');
+const { logger } = require('@senecacdot/satellite');
```

Comment:
I wouldn't bother with the dep on Satellite for this, use `console.log()` and friends.